Utilites/BookingUtilites.cs

```diff
@@ -0,0 +1,74 @@
+﻿
+using Airport_Ticket_Booking_System.Bookings;
+using Airport_Ticket_Booking_System.Flights;
+using Airport_Ticket_Booking_System.Users;
+
+namespace Airport_Ticket_Booking_System.Utilites;
```

Comment:
typo
```suggestion
namespace Airport_Ticket_Booking_System.Utilities;
```


I would like to suggest renaming this to represent repositories. 

our app structure should be as follows:
Model: a blueprint for your custom and business types. 
Repository: a layer to do the communication between with he storage directly.
Service: the layer where you put your business logic. 
And Presentation: the layer that interacts with the user.